### ISSUE TITLE
gesture recognizer android error 

### ISSUE THREAD

**Issue by Rawabi20** [googlesamples/mediapipe#276]
When I run the example in android studio I got this:
FATAL EXCEPTION: pool-2-thread-1
                                                                                                    Process: com.google.mediapipe.examples.gesturerecognizer, PID: 5543
                                                                                                    java.lang.UnsatisfiedLinkError: dlopen failed: library "libmediapipe_tasks_vision_jni.so" not found
                                                                                                    	at java.lang.Runtime.loadLibrary0(Runtime.java:1082)
                                                                                                    	at java.lang.Runtime.loadLibrary0(Runtime.java:1003)
                                                                                                    	at java.lang.System.loadLibrary(System.java:1661)
                                                                                                    	at com.google.mediapipe.tasks.vision.core.BaseVisionTaskApi.<clinit>(BaseVisionTaskApi.java:37)
                                                                                                    	at com.google.mediapipe.examples.gesturerecognizer.GestureRecognizerHelper.setupGestureRecognizer(GestureRecognizerHelper.kt:95)
                                                                                                    	at com.google.mediapipe.examples.gesturerecognizer.GestureRecognizerHelper.<init>(GestureRecognizerHelper.kt:50)
                                                                                                    	at com.google.mediapipe.examples.gesturerecognizer.fragment.CameraFragment.onViewCreated$lambda-4(CameraFragment.kt:147)
                                                                                                    	at com.google.mediapipe.examples.gesturerecognizer.fragment.CameraFragment.$r8$lambda$xiZI6LDAjMBw-J7vyrjSe_CLWo0(Unknown Source:0)
                                                                                                    	at com.google.mediapipe.examples.gesturerecognizer.fragment.CameraFragment$$ExternalSyntheticLambda13.run(Unknown Source:2)

**Comment by marceper09** [comment]
I have the same problem.

**Comment by PaulTR** [comment]
What kind of device are you installing on?

**Comment by Rawabi20** [comment]
> What kind of device are you installing on?

windows, and the emulator is 3a

**Comment by PaulTR** [comment]
Ah I see. So for the best results you're going to want to use a real physical device. Have you seen this issue on a test device?

**Comment by Rawabi20** [comment]
> est device

Actually I don't have physical device, so I haven't  try it....
There is no way to run it on emulator?

**Comment by Rawabi20** [comment]
> Ah I see. So for the best results you're going to want to use a real physical device. Have you seen this issue on a test device?

Actually I don't have physical device, so I haven't try it....
There is no way to run it on emulator?

**Comment by PaulTR** [comment]
For an emulator try one of the later OS versions (I just ran on a pixel 6 emulator, but I'm also on a mac). Other than that it might just not work out for the emulator, unfortunately. ML is pretty tied in with hardware for how it works.

**Comment by Rawabi20** [comment]
> For an emulator try one of the later OS versions (I just ran on a pixel 6 emulator, but I'm also on a mac). Other than that it might just not work out for the emulator, unfortunately. ML is pretty tied in with hardware for how it works.

I got an old device to try the code on it and I got this error:
The application could not be installed: INSTALL_FAILED_OLDER_SDK

**Comment by PaulTR** [comment]
So based on that error message, what SDK version are you using on the device? There's a minimum for the library :) https://github.com/googlesamples/mediapipe/blob/main/examples/gesture_recognizer/android/app/build.gradle#L29

**Comment by StormRunner06106** [comment]
Is there a no way to run media object detection sample on the Android G emulator?

**Comment by StormRunner06106** [comment]
Hello, I have same problem, so how can i solve this problem in Pixel 4XL or other devices?

**Comment by JenyaKirmiza** [comment]
same issue on production

<img width="548" alt="Image" src="https://github.com/user-attachments/assets/b71dfdbc-c7d2-43b2-b77b-958753da400b" />
<img width="548" alt="Image" src="https://github.com/user-attachments/assets/2ed36c12-de3f-4aa5-a448-7addc712a12b" />

**Comment by JenyaKirmiza** [comment]
after updating to 0.10.26 the issue reappeared 

```
    Fatal Exception: java.lang.UnsatisfiedLinkError: dlopen failed: library "libmediapipe_tasks_vision_jni.so" not found
       at java.lang.Runtime.loadLibrary0(Runtime.java:1090)
       at java.lang.Runtime.loadLibrary0(Runtime.java:1012)
       at java.lang.System.loadLibrary(System.java:1765)
       at com.google.mediapipe.tasks.vision.gesturerecognizer.GestureRecognizer.<clinit>(GestureRecognizer.java:97)
       at com.eden_android.verification.camera.GestureRecognizerHelper.setupGestureRecognizer(GestureRecognizerHelper.kt:70)
       at com.eden_android.verification.camera.GestureRecognizerHelper.<init>(GestureRecognizerHelper.kt:29)
       at com.eden_android.verification.camera.CameraFragment.onViewCreated$lambda$11(CameraFragment.java:236)
       at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1156)
       at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:651)
       at java.lang.Thread.run(Thread.java:1119)
```

**Comment by abbassabetinezhad-seon** [comment]
Apparently, this issue which was reappeared in 0.10.26 is now fixed on 0.10.26.1.
